subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/fixture/GradleBuildExperimentSpec.groovy

```diff
@@ -88,6 +91,11 @@ class GradleBuildExperimentSpec extends BuildExperimentSpec {
             this
         }
 
+        GradleBuilder measuredBuildOperations(List<String> measuredBuildOperations) {
+            this.measuredBuildOperations = ImmutableList.copyOf(measuredBuildOperations)
+            this
+        }
```

Comment:
These will be ignored for "traditional" performance tests, right? We should probably fail if someone tried to set it in that case.

Reply:
Currently, you can't set those operations on the runner itself, since there are two separate classes: `GradleProfilerCrossVersionPerformanceTestRunner` and `CrossVersionPerformanceTestRunner`.

`buildExperimentListeners` and `invocationCustomizers` can only be specified on `CrossVersionPerformanceTestRunner`, while `buildMutators` and `measuredBuildOperations` can only be specified on `GradleProfilerCrossVersionPerformanceTestRunner`.

I think that should be enough already.